app/controllers/api/v1/tokens_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class TokensController < Doorkeeper::TokensController
+      include ErrorHandlerConcern
+
+      def revoke
```

Comment:
Why you need to override `revoke` action? Usually doorkeeper return empty response which if the revoke success. Or you want to show some JSON API response?

Reply:
Yes @junan, empty reponse is not compliant with JSON API standard :P That's the cleanest way we found with Ankit ^^